onnxscript/rewriter/onnxruntime/__init__.py

```diff
@@ -27,6 +28,7 @@ def rewrite(
     /,
     function_rules: list[type[function_rule.FunctionRewriteRule]] | None = None,
     pattern_rules: list[pattern.RewriteRule] | None = None,
+    convert_bloat16_to_float16: bool = False,
```

Comment:
I am conscious that the interface is changed. I personally don't think this api should be modified

Reply:
I am not sure how this tool is going to be used actually. @thiagocrepaldi might have an idea.

Reply:
But this is indeed not a long term argument. Maybe we should hide it.

Reply:
Yes, we need a better strategy here ... especially for utils like this. How is this intended to be used? Do we expect the torch exporter to invoke this at export time, or will it be done by the users separately?

Here, one option is to introduce the notion of a pass (which is essentially an in-place transformer of an ir.Model), and expose this conversion-utility as a pass, and allow the input to simply be a list of passes. We should be able to create a pass out of a list of rewrite-rules (of different kinds). Eventually, this and the optimizer API should be unified.

Reply:
For now, exposing as a separate utility called by user might be okay, rather than changing this API frequently?